### DISCUSSION TITLE
为什么设置的分流规则全都不生效？？

### DISCUSSION THREAD

**Discussion by daowaiyaoyao** [vernesong/OpenClash#3742] [Q&A]
系统是istoreos最新版
opencalsh最新版
问题如下：比如我默认线路是香港，然后设置油管走美国，openai也走美国，但是我发现油管跟openai最终全是走的香港，咋办呢？

**Comment by ghost** [comment]
去控制面板看连接页
有类似`a/z/香港`意思就是匹配到a组被转到z再通过香港连接
具体的看你配置文件咋写的。
